### ISSUE TITLE
stuck when running the codes

### ISSUE THREAD

**Issue by taozhuo2000** [solicucu/D3G#3]
model = torch.nn.parallel.DistributedDataParallel(             
            model, device_ids=[local_rank], output_device=local_rank,
            # this should be removed if we update BatchNorm stats
            broadcast_buffers=False, find_unused_parameters=True
        )

the experiment is stucked on this step, no matter how I try to fix it. 
Please help me,  thank you!
